client/src/App.jsx

```diff
@@ -1,23 +1,32 @@
-import { useContext } from 'react'
-import './App.css'
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { useContext } from 'react';
+import './App.css';
 
 // Component Imports
-import NavBar from './assets/components/layout/NavBar'
+import NavBar from './assets/components/layout/NavBar';
 
 // Context Imports
 import ThemeContext from './assets/context/ThemeContext';
+import LoginPage from './assets/pages/LoginPage';
+import SignPage from './assets/pages/SignPage';
 
 function App() {
-
   const { darkMode } = useContext(ThemeContext);
 
   return (
-    <main className='w-full min-h-screen' data-theme={darkMode ? "dim" : "nord"}>
-      <div className='max-w-[2000px] mx-auto min-h-screen px-2 py-4 bg-base-200'>
-        <NavBar />
-      </div>
-    </main>
-  )
+    <Router>
+      <main className='w-full min-h-screen' data-theme={darkMode ? "dim" : "nord"}>
+        <div className='max-w-[2000px] mx-auto min-h-screen px-2 py-4 bg-base-200'>
+          <NavBar />
+          <Routes>
+            <Route path="/" element={<SignPage />} />
```

Comment:
I like this idea, as soon as a new person comes to the page, they should be greeted by signing up first. Or we can find an alternative, I will think of other options.

Please leave this as originally intended as we will implement this after your user auth is completed. Root / should return index page. If needed, create a Home.jsx in components, temporarily.